Model/Maker/UiComponentGrid.php

```diff
@@ -0,0 +1,68 @@
+<?php
+declare(strict_types=1);
+
+namespace Ctasca\MageBundle\Model\Maker;
+
+use Ctasca\MageBundle\Api\MakerUiComponentInterface;
+use Ctasca\MageBundle\Api\MakerUiComponentXmlInterface;
+use Ctasca\MageBundle\Console\Question\Prompt\Validator as QuestionValidator;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class UiComponentGrid extends AbstractMaker implements MakerUiComponentXmlInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function make(InputInterface $input, OutputInterface $output): void
+    {
+        $question = $this->makeModuleNameQuestion();
+        $moduleName = $this->questionHelper->ask($input, $output, $question);
+        try {
+            /** @var \Ctasca\MageBundle\Model\Template\Locator $templateLocator */
+            list($templateLocator,)  = $this->locateTemplateDirectory(self::UI_COMPONENT_TEMPLATES_DIR);
+            $question = $this->makeGridNamespaceQuestion(
```

Comment:
Maybe add an area choice question here before this question? Either frontend or adminhtml...

Reply:
al momento l'xml creato ha senso solo per l'area adminhtml, quindi mi sembrava inutile aggiungere la question